libs/partners/openai/tests/integration_tests/chat_models/test_azure_standard.py

```diff
@@ -9,27 +9,39 @@
 
 from langchain_openai import AzureChatOpenAI
 
-OPENAI_API_VERSION = os.environ.get("AZURE_OPENAI_API_VERSION", "")
```

Comment:
Ran OpenAI integration tests on this branch and got failures related to env vars: https://github.com/langchain-ai/langchain/actions/runs/10924401318/job/30323353395

Also getting some failures here in json mode on master, suspect unrelated. Unable to reproduce so far.
```
FAILED tests/integration_tests/chat_models/test_azure.py::test_json_mode_async - openai.LengthFinishReasonError: Could not parse response content as the length limit was reached - CompletionUsage(completion_tokens=10, prompt_tokens=18, total_tokens=28, completion_tokens_details=None)
```

Reply:
huh thought id set that env var, will double check